### PR TITLE
Remove "auto" folder when regenarating non-sys crate

### LINKED ISSUE #1269: generator script should remove src/auto before regen
Otherwise removed files (no longer generated ones) are kept there and you might end up forgetting to drop them

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes  #1269.

### CODE REVIEW

File: generator.py [hunk 1]
```diff
@@ -136,6 +137,8 @@ def regen_crates(path, conf):
         else:
             if is_sys_crate:
                 args.extend(["-m", "sys"])
+            else:
+                shutil.rmtree(path.parent / "src/auto")
```
Comment (sdroege) [inline generator.py:141]: Does this follow symlinks? I hope not

**Comment by MarijnS95** [comment]
Can't we do this inside `gir` instead?

**Comment by GuillaumeGomez** [comment]
I guess we could. What do you think about it @sdroege ?

**Comment by sdroege** [comment]
I prefer in the script a bit over the other solution, but whatever you prefer

**Comment by GuillaumeGomez** [comment]
And you @bilelmoussaoui ?

**Comment by bilelmoussaoui** [comment]
In gir it would make more sense I guess

**Pull request merged**